target/bin/sedfile

```diff
@@ -11,19 +11,18 @@
 
 set -ueo pipefail
 
-HASHTOOL="sha1sum"
+HASHTOOL='sha1sum'
 SKIP_ERROR=0
 
 if [[ $# -lt 3 ]]
 then
-  echo "Error:  At least, three parameters must be given."
-  echo "Syntax: sedfile -i <replace/delete operation> <file>"
-  echo
+  echo 'At least three parameters must be given'
+  echo -e "Usage: ${0} -i <replace/delete operation> <file>\n"
```

Comment:
Personal gusto. I found the previous "3 echo solution" more readable, don't you?

Reply:
If we're using `_log` as said above, the first message will become a log message and the `Usage: ...` will be outsourced in a function to be coherent with all the other scripts.

Reply:
I just noticed, we should not use `log` or any other helper in `sedfile` because the script is copied earlier in Dockerfile and also used early, which means that the scripts under `/usr/local/bin/helpers/` are not yet there. I will adjust accordingly.